File: beginner/java/2313.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int a, b, c;

        a = scanner.nextInt();
        b = scanner.nextInt();
        c = scanner.nextInt();

        if(b + c > a && a + c > b && a + b > c){
            System.out.printf("Valido-");
            if (a == b && b == c) {
                System.out.printf("Equilatero%n");
            } else if (a == b || b == c || a == c) {
                System.out.printf("Isoceles%n");
            } else {
                System.out.printf("Escaleno%n");
            }

            if (a * a == b * b + c * c || b * b == a * a + c * c || c * c == a * a + b * b) {
                System.out.printf("Retangulo: S%n");
            } else {
                System.out.printf("Retangulo: N%n");
            }
        }
        else{
            System.out.printf("Invalido%n");
        }
        scanner.close();
    }
}

File: beginner/c/2313.c
#include <stdio.h>

int main() {
    
    int a, b, c;

    scanf("%d %d %d", &a, &b, &c);

    if(b + c > a && a + c > b && a + b > c){
        printf("Valido-");
        if (a == b && b == c) {
            printf("Equilatero\n");
        } else if (a == b || b == c || a == c) {
            printf("Isoceles\n");
        } else {
            printf("Escaleno\n");
        }

        if (a * a == b * b + c * c || b * b == a * a + c * c || c * c == a * a + b * b) {
            printf("Retangulo: S\n");
        } else {
            printf("Retangulo: N\n");
        }
    }
    else{
        printf("Invalido\n");
    }

    return 0;
}